src/features/spa/aggregate/serializer.js

```diff
@@ -128,6 +127,11 @@ export class Serializer extends SharedContext {
             nullable(node.dt && node.dt.traceId, addString, true) +
             nullable(node.dt && node.dt.timestamp, numeric, false)
           )
+
+          // add params.gql here
+          var ajaxAttrParts = addCustomAttributes({ ...(params.gql || {}) }, addString)
+          children = children.concat(ajaxAttrParts)
+          attrCount = ajaxAttrParts.length
```

Comment:
I don't _think_ ajax nodes rn can have children, do you know if this is true and/or this requires consumer parsing?

Reply:
Ajax nodes use children to report custom attributes today.  Will validate via `experiment` publish in NR1 once the test suite issues have been resolved